tests/mock_crust/churn.rs

```diff
@@ -181,20 +219,26 @@ fn request_during_churn_node_to_group() {
         let data_id = data.identifier();
         let message_id = MessageId::new();
 
+        let except_index = added_index.unwrap_or(nodes.len());
+        let quorum_before = quorum(nodes.iter()
+            .enumerate()
+            .filter(|&(i, n)|
+                i != except_index && n.routing_table().is_recipient(&dst.to_destination()))
```

Comment:
Maybe it would be better to exclude the node in some other way, so that we could reuse `target_group()` here - for example, pass the nodes without the one to be excluded to `target_group()`.